app/routes/explore.js

```diff
@@ -1,14 +1,33 @@
 import Ember from 'ember';
 import moment from 'moment';
 
-const { Route } = Ember;
+const { Route, RSVP } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Explore');
   },
 
   model() {
-    return this.store.query('event', { end_time_gt: moment.utc().format('YYYY-MM-DDTHH:mm:ss'), state: 'Published' });
+    return RSVP.hash({
+      events: this.store.query('event', {
+        include : 'event-topic,event-sub-topic,event-type',
```

Comment:
Don't include. With the ID ember should automatically pick right one when needed since we load types, topics and sub-topics in this page anyway.